scss/utilities/mixins/molecules/_footer-logo.scss

```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
```

Comment:
This does not need to be its own component. I don't see the value it ins re-use.